lib/lib_rss.php

```diff
@@ -578,3 +578,21 @@ function validateShortcutList($shortcuts) {
 	}
 	return $shortcuts_ok;
 }
+
+function errorMessage($errorTitle, $error = '') {
+	// Prevent empty <h2> tags by checking if error isn't empty first
+	if ('' !== $error) {
+		$error = "<h2>{$error}</h2>";
+	}
+	return <<<MSG
+	<h1>{$errorTitle}</h1>
+	{$error}
+	<pre>
+	See log files for the feed update service "journalctl -xeu freshrss".
+	If you do not use a service to update the feeds, run the script manually "/usr/sbin/php /usr/share/webapps/freshrss/app/actualize_script.php".
+	If you get no hints from there, see logs for your webserver i.e. "journalctl -xeu nginx".
+	If you are using php-fpm, also check logs for it "journalctl -xeu php-fpm".
+	</pre>
+	<b>Note</b>: Change paths and service names as necessary depending on where and what things are installed. Additional logs may be in "/var/log".
```

Comment:
```suggestion
	<h2>Common problems</h2>
	<p>A typical problem leading to this message is wrong file permissions in the <kbd>./FreshRSS/data/</kbd> folder so make sure the Web server can write there and in sub-directories.</p>
	<h2>Common locations for additional logs</h2>
	<p><strong>N.B.:</strong> Adapt names and paths according to your local setup.</p>	
	<ul>
	<li>If using Docker: <kbd>docker logs -f freshrss</kbd></li>
	<li>To check Web server logs on a Linux system using systemd; <kbd>journalctl -xeu apache2</kbd>
	and if you are using php-fpm: <kbd>journalctl -xeu php-fpm</kbd></li>
	<li>Otherwise, Web server logs are typically located in <kbd>/var/log/apache2/</kbd> or similar</li>
	<li>System logs may also contain relevant information in <kbd>/var/log/syslog</kbd></li>
	</ul>
	<p>More logs can be generated by enabling <code>'environment' => 'development',</code> in <kbd>./FreshRSS/data/config.php</kbd></p>
```

A user would not see this message when using cron, so I not not think the cron/service is very relevant here.
We could link to a troubleshooting page with better guidance.

Reply:
Something like samp or code would be more appropriate than kbd, wouldn't it?

Reply:
>so I do not think the cron/service is very relevant here

It is relevant. In my case, I've had an issue where I didn't have pdo enabled/installed as I was donwgrading from PHP 8 to 7, and the only way to figure it out was to look at the actualize_script logs which pointed it out.

```
/usr/sbin/php /usr/share/webapps/freshrss/app/actualize_script.php
FreshRSS starting feeds actualization at 2021-01-30T21:08:25+00:00
FreshRSS install failed requirements:
• pdo-pgsql
• ok
```

Reply:
I used your comment but replaced the kbd tags with code tags and kept the actualize script note in